src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -305,6 +305,12 @@ protected static async Task<string> DownloadPackageFile(string installerUrl)
                     return null;
                 }
 
+                if (e is TaskCanceledException)
```

Comment:
For clarity can you make these all if-else blocks?

Reply:
Added else-if blocks for clarify